Source/GR32_System.pas

```diff
@@ -226,7 +226,7 @@ function GetProcessorCount: Cardinal;
     ($800000,  $400000, $2000000, $4000000, $80000000, $40000000);
     {ciMMX  ,  ciEMMX,  ciSSE   , ciSSE2  , ci3DNow ,  ci3DNowExt}
 
-function CPUID_Available: Boolean;
+function CPUID_Available: Boolean; assembler;

```

Comment:
Needs {$IFDEF FPC}...{$ENDIF}

Reply:
If I'm right in thinking, Delphi doesn't actually care if 'assembler' is there or not, but will do.